Vagrantfile

```diff
@@ -28,9 +28,7 @@ Vagrant.configure(2) do |config|
   end
 
   # Sync the sources folder with the machine.
-  # For Windows `nfs` is preffered due to poor performance of default settings.
-  synced_folder_type = Vagrant::Util::Platform.windows? ? 'nfs' : nil
-  config.vm.synced_folder '.', '/vagrant', type: synced_folder_type
+  config.vm.synced_folder '.', '/vagrant', type: :nfs
```

Comment:
This is not a good choice, nfs was a windows specific workaround, and it makes life harder on linux, requiring nfsd to be started and a root password any time the vm is woken up.